pkg/openshift/certgen/templates/templates.go

```diff
@@ -1,4 +1,4 @@
 //go:generate go get -u github.com/go-bindata/go-bindata/...
-//go:generate go-bindata -pkg templates master/... node/...
+//go:generate go-bindata -modtime 1 -pkg templates master/... node/...
```

Comment:
why are you overriding the mod time stamp here?

Reply:
Git doesn't persist timestamps so if `-modtime 1` is not included here it results in a lot of churn in commits.

Reply:
(origin uses -nometadata instead, it turns out)